source/osmaxx/urls.py

```diff
@@ -10,4 +10,6 @@
 
     url(r'^excerptexport/', include('excerptexport.urls', namespace='excerptexport')),
     url(r'^admin/', include(admin.site.urls)),
+
+    url('', include('social.apps.django_app.urls', namespace='social')),
```

Comment:
I commented your line, because you had a 50% decision to use strings or variables ;-) and you refactored all urls in the other file. so we should do it here too...